Source/DiabloUI/diabloui.cpp

```diff
@@ -453,9 +453,11 @@ void UiHandleEvents(SDL_Event *event)
 			// but may now become visible.
 			DoReinitializeHardwareCursor();
 		} else if (event->window.event == SDL_WINDOWEVENT_FOCUS_LOST) {
-			music_mute();

```

Comment:
Will the above boolean assignment to `gbActive` conflict with this PR at all?

Reply:
Not sure I'm following that question

Reply:
gbActive is set false if the window is minimized or hidden. So I'm wondering if this will conflict with the purpose of this PR. Unable to look up what gbActive is used for right now.

Reply:
The tests for gbActive are:

1) `Source/menu.cpp` - in the main menu loop - determines whether or not to play intro

2) `Source/engine/render/scrollrt.cpp` -  Deciding whether or not to draw the FPS, and secondarily, to draw main 

3) `Source/engine/dx.cpp` - Deciding whether or not to limit frame rate

4) `Source/DiabloUI/dialogs.cpp` - in UiOkDialog - deciding whether or not to show a cursor or message box (I think) 

So I don't think so.  Might want to ask whoever wrote that though.